### PR TITLE
Add separate examples for arithmetic operators

### PR DESCRIPTION
**Pull request by wbamberg**
Adding separate examples for the arithmetic operators, as part of https://github.com/mdn/stumptown-content/issues/411.

One thing I'm not crazy about here is having unary-plus be a long example, but it's hard to fit into 13 lines and cover the main points.

### CODE REVIEW

File: live-examples/js-examples/expressions/expressions-addition.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+console.log(2 + 2);
+// expected result: 4
+
+console.log(2 + true);
+// expected result: 3
+
+console.log('hello ' + 'everyone');
+// expected result: "hello everyone"
+
+console.log(`${2001  }: A Space Odyssey`);
```
[the remark this answers is not in the document]
Comment (wbamberg) [reply live-examples/js-examples/expressions/expressions-addition.js]: This was supposed to show string concatenation using `+`, but I ran `npm lint:js:fix` and it changed it to use template literals :). I've changed it back and added an eslint exception for this example. Thanks for spotting this Florian!

**Comment by wbamberg** [comment]
> I wondered if any of the things shown, like console.log(+true); and console.log(+false); are esoteric and can be left out, but I'm not sure.

The thing is, even without those we're at 14 lines, and the others all seem quite important.

**Comment by wbamberg** [comment]
You could get 3 lines back by doing:

```js
console.log(+(1));
// expected output: 1

console.log(+(-1));
// expected output: -1

console.log(+'');
// expected output: 0

console.log(+'hello');
// expected output: NaN
```

...but those `(+(-1))` bits are rather hard to read.

File: live-examples/js-examples/expressions/expressions-addition.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Elchi3) [reply live-examples/js-examples/expressions/expressions-addition.js]: Aha! I wonder if more code is getting rewritten :) But this was an obvious thing to spot.

[the remark this answers is not in the document]
Comment (wbamberg) [reply live-examples/js-examples/expressions/expressions-addition.js]: It's definitely a lesson to run just `lint:js` first, and check whether the errors are really things we want to fix.

**Pull request merged**